Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: format docker
 
 BINARY=vm-discovery
-DOCKER_REPO=jackyzz
+DOCKER_REPO=linggg
```

Comment:
@brian-avery can find a place for these images that isn't an individual contributor repo?

Reply:
I created a docker organization -- `istioecosystem`. Lets use that instead.

Reply:
I tried to push to istioecosystem but got access denied error. Could you add me to the organization @brian-avery ? My docker id is `linggg`.

```
docker push istioecosystem/vm-discovery:latest
The push refers to repository [docker.io/istioecosystem/vm-discovery]
b01ed0cf0266: Preparing 
5f08512fd434: Preparing 
c7bb31fc0e08: Preparing 
50858308da3d: Preparing 
denied: requested access to the resource is denied
```

Reply:
Let me know when and I can build and upload it

Reply:
It's ready to build from master. Thanks @brian-avery

Reply:
Thanks and you're welcome :) https://hub.docker.com/r/istioecosystem/vm-discovery/tags